Detectors/TRD/base/include/TRDBase/Digit.h

```diff
@@ -60,22 +60,22 @@ class Digit
   void setDetector(int det) { mDetector = det; }
   void setADC(ArrayADC const& adc) { mADC = adc; }
   // Get methods
-  int getDetector() const { return mROB; }
+  int getDetector() const { return mDetector; }
   int getRow() const { return FeeParam::getPadRowFromMCM(mROB, mMCM); }
   int getPad() const { return FeeParam::getPadColFromADC(mROB, mMCM, mChannel); }
   int getROB() const { return mROB; }
   int getMCM() const { return mMCM; }
   int getChannel() const { return mChannel; }
-  int isSharedDigit();
+  bool isSharedDigit();
```

Comment:
Please add a comment here, or maybe even rename the function. "Shared" could be ambiguous: it could also refer to pads 2, 18 or 19.

Reply:
Fair enough, its confusing as hell. So you mean something like isSharedChannel?

Reply:
The reason for the name is that you can query if the digit in question (assigned to an MCM channel) is in fact shared. It is then up the user calling to figure out where. It could possibly return the mcm/rob of the associated shared channel?